supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -152,23 +161,68 @@ async def create_new_session(
         session_data: Union[None, Dict[str, Any]],
         user_context: Dict[str, Any],
     ) -> SessionContainer:
+        log_debug_message("createNewSession: Started")
+        output_transfer_method = self.config.get_token_transfer_method(
+            request, True, user_context
+        )
+        if output_transfer_method == "any":
+            output_transfer_method = "header"
+
+        log_debug_message(
+            "createNewSession: using transfer method %s", output_transfer_method
+        )
+
+        # TODO: Move TLD checking logic here
+
+        disable_anti_csrf = output_transfer_method == "header"
+
         session = await session_functions.create_new_session(
-            self, user_id, access_token_payload, session_data
+            self,
+            user_id,
+            disable_anti_csrf,
+            access_token_payload,
+            session_data,
         )
+
+        # access_tokens: Dict[TokenTransferMethod, str] = {} # FIXME: type
+
+        session_methods_to_call: List[Callable[[Any], Any]] = []
```

Comment:
Pleas be explicit about the typings. This function should take in the baseResponse object and return void.